internal/cephfs/controllerserver.go

```diff
@@ -188,7 +188,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 			// explictly
 			err = resizeVolume(ctx, volOptions, cr, volumeID(vID.FsSubvolName), volOptions.Size)
 			if err != nil {
-				if purgeErr := purgeVolume(ctx, volumeID(vID.FsSubvolName), cr, volOptions, false); err != nil {
+				if purgeErr := purgeVolume(ctx, volumeID(vID.FsSubvolName), cr, volOptions, false); purgeErr != nil {
```

Comment:
These issues can quickly happen. It is much easier to spot when the error assignment and if condition are on two lines. Could you please improve the maintainability that way?

Also, what is the (user visible) problem that this solves? It needs to be added in the commit message.

Reply:
Done.